release-notes/fleet-elastic-agent/deprecations.md

```diff
@@ -12,10 +12,18 @@ Review the deprecated functionality for {{fleet}} and {{agent}}. While deprecati
 % ::::{dropdown} Deprecation title
 % Description of the deprecation.
 % For more information, check [PR #](PR link).
-% **Impact**<br> Impact of deprecation. 
+% **Impact**<br> Impact of deprecation.
 % **Action**<br> Steps for mitigating deprecation impact.
 % ::::
 
-% ## 9.0.0 [fleet-elastic-agent-900-deprecations]
+## 9.1.0 [fleet-elastic-agent-9.1.0-deprecations]
+
+::::{dropdown} Deprecate the elasticinframetrics processor.
+The elasticinframetrics processor is no longer necessary, is deprecated in 9.1.0, and is scheduled for removal in 9.2.0.
+
+For more information, check [#8659]({{agent-pull}}8659).
+::::
```

Comment:
@theletterf this was included in the generated release notes in https://github.com/elastic/elastic-agent/pull/9160. Does it make sense for this to be here? I think it's related to the EDOT Collector.

Reply:
My suggestion:

```suggestion
::::{dropdown} Deprecate the elasticinframetrics processor.
The elasticinframetrics processor is no longer necessary and is deprecated in EDOT Collector 9.1.0. It is scheduled for removal in 9.2.0.

For more information, check [#8659]({{agent-pull}}8659).
::::
```

When we'll have dedicated Collector relnotes, I guess we'll move them there, but AFAIK Collector relnotes items are being added to Agent relnotes for now, no?